src/utils/date.ts

```diff
@@ -0,0 +1,25 @@
+import { format } from 'date-fns';
+
```

Comment:
export DATE_FORMAT = 'yyyy-mm-dd'

I was only looking for a simple:

export const formatDate = (date: Date, format = DATE_FORMAT) {
  return (date, format);
}

But what is here is good, we can though remove `includeTime` if we pass in the data format and leafve date as string | DAte

Reply:
Made the changes